modules/tinymce/src/plugins/table/main/ts/ui/CellDialog.ts

```diff
@@ -115,18 +117,22 @@ const applyCellData = (editor: Editor, cells: SugarElement<HTMLTableCellElement>
       });
     });
 
+    const cellTypeModified = Obj.has(modifiedData, 'celltype');
```

Comment:
`scope` is not a style change either, so might need to expand on this a bit? Not sure if we want to call it a structural change either; I'd be happy to not fire either for it atm and make another Jira to figure that out?

Reply:
While I'm looking into it, noting that rows have `scope` too and tables have `caption` which is definitely structural. Will check the PR as I go for those.

Reply:
I'm torn when it comes to `scope`. It's definitely not structural, and it doesn't change the style either. We could just go `false` for both?